Examples/VisionExamples/Code Snippets/GettingStartedViewController.swift

```diff
@@ -1,57 +1,55 @@
-/**
- This file shows basic Vision SDK configuration steps.
- */
+// This file shows basic Vision SDK configuration steps.
```

Comment:
I'd leave the current formatting of example headers. Didn't notice that in PR to master.

Reply:
Formally, it was an empty doc comment which is incorrect.

I propose to merge the current PR anyway to sync the state, but we can use multiline comment
```swift
/*
This file shows basic Vision SDK configuration steps.
*/
```

here and in the rest of places later (while we're merging examples/code snippets).

Does that make sense?

Reply:
I'm ok to merge it like this and adjust with other example PRs. 
As for the comment look, we have already chosen the style for such comments (all the example files follow it), do you suggest to change it?